android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/RunFragment.kt

```diff
@@ -66,24 +101,31 @@ class RunFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
         val chronometer = view.findViewById<Chronometer>(R.id.chronometer)
         val runBtn = view.findViewById<Button>(R.id.run_btn)
```

Comment:
Now that you have the binding object, you no longer need to perform findViewById. These views should be attributes of the binder like `binding.chronometer`.